site/content/docs/latest/tutorials/managing-package-repositories.md

```diff
@@ -0,0 +1,262 @@
+- **Private repositories** need to authorize Kubeapps to access the repository.
+
+Open the **Authorization** tab that displays the following authorization options:
+
+- `None`: select to configure the Bitnami Application Catalog (or any other public repo accessible without authentication).
+- `Basic Auth`: specify the username and password for the repository.
+- `Bearer Token`: specify a token (previously created) to authenticate within the repository.
+- `Docker Registry Credentials`: specify the server, username, password and email to authorize access to the registry. It is possible to use the same auth credentials for the container registry (or manually set different credentials for the container registry).
+- `Custom Authorization Header`: specify a raw authorization header to authorize access to the registry.
+- `SSH based Authentication`: specify a raw SSH know host and private key to authorize access to the registry.
+- `TLS-based Authentication`: specify a raw TLS cert and key to authorize access to the registry.
+- `Opaque-based Authentication`: specify a raw opaque data (JSON) to authorize access to the registry.
+
+![Authentication options](../img/package-repository/package-repository-authentication.png)
+
+**NOTE**: Any authorization method allows to use an existing secret by providing the secret values (`secret name`). This option needs to have Kubeapps configured to use secrets instead of values.
+
+### Filter applications
+
+> **NOTE**: Feature only available for Helm (not implemented for Helm via Flux or Carvel yet).
+
+It is possible to limit the number of packages that Kubeapps exposes to the users of a Package Repository. For doing so, edit the information in the **Filtering** tab:
+
+- `List of Repositories`: introduce the list of OCI repositories that will be available in Kubeapps.
+- `Filter Applications`: introduce the list of applications to be included (by default) or excluded (if exclude packages selected).
+- `Exclude Packages`: if marked, all the applications matching the given names will be excluded.
+- `Regular Expression`: if marked, rather than treating the input as a comma-separated list of application names, it will be treated as a [PCRE regex](https://stedolan.github.io/jq/manual/#RegularexpressionsPCRE). This can be used when the name of the application is not known or when you want to filter different applications with the same name.
+
+![AppRepository with filter](../img/package-repository/apprepo-form-filter.png)
+
+#### Advanced filtering
+
+> **NOTE**: This is not supported by the Kubeapps Dashboard.
+
+In case you want to add a custom filter, based on a metadata field different than the name, it's possible to specify a [jq](https://stedolan.github.io/jq/) query to filter applications. This is only available when manually creating the AppRepository manifest. This example filters applications that contain "Bitnami" as one of the maintainers:
```

Comment:
> This is only available when manually creating the AppRepository manifest

Maybe we should specify that it only applies to the direct Helm plugin?

Reply:
+1